### ISSUE TITLE
Error when building with PrivateFramework

### ISSUE THREAD

**Issue by haoict** [angelXwind/simject#60]
In `Makefile` I set:
```
ARCHS = x86_64
TARGET = simulator:clang:13.6:13.0
...
$(TWEAK_NAME)_PRIVATE_FRAMEWORKS = Preferences
```

Then `make` throws error
```
ld: warning: directory not found for option '-F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/PrivateFrameworks'
ld: framework not found Preference
```

I understand that the sdk it's using doesn't have PrivateFramework.

I can get patched sdk from https://github.com/xybp888/iOS-SDKs and built successfully with `TARGET = iphone:clang:13.6:13.0`. But it's only for iPhoneOS, not iPhoneSimulator

Any workaround for this?

**Comment by opa334** [comment]
Use the patched 13.7 sdk from my repo: https://github.com/opa334/sdks

**Comment by haoict** [comment]
Thanks, let me check and comment back soon
